README.md

```diff
@@ -118,29 +118,31 @@ You'll need Docker on your machine for local development, otherwise Docker will
+    - Select the option "My Application is stored on GitHub"
+    - Enter a new GitHub token name (GitHub username), the repository name, and the full commit id of the latest commit in the branch you're looking to deploy (can be found in the commit's url)
+    - Every other option can be left as is. Click Create Deployment, and you're done!
+
+3. [Using the AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/deploy/create-deployment.html). You'll only ever do this in the rare case that the GitHub/CodeDeploy integration is broken.
+
+At time of writing, the AWS environment is not set up to have review apps.
 
 ## Details about how databases are connected
 The application uses two databases:
 1. A Postgres instance on AWS RDS. This is the application's main database that we write to.
+    - The RDS instance is named `asset-dashboard`. We created two databases within the instance: `production` and `staging`.
+    - Each app for the CIP dashboard in EC2 is configured to connect with the corresponding environment's database.
+    - When we were hosting this on Heroku, the RDS security group had to be configured to accept connections from the application on Heroku. Since Heroku doesn't have static IP addresses, the [quotagaurd static add-on](https://elements.heroku.com/addons/quotaguardstatic) helped establish a connection with the remote database. For details on how that worked, see [PR #91](https://github.com/fpdcc/ccfp-asset-dashboard/pull/91) and [PR #70](https://github.com/fpdcc/ccfp-asset-dashboard/pull/70), as well as discussions in issues [#59](https://github.com/fpdcc/ccfp-asset-dashboard/issues/59) and [#60](https://github.com/fpdcc/ccfp-asset-dashboard/issues/60).
 
-The RDS instance is named `ccfp-asset-dashboard`. We created three databases within the instance:
-1. `production`
-2. `staging`
-3. `review`
-
-Each app in the Heroku pipeline is configured to connect with the corresponding environment's database.
-
-The RDS security group is configured to accept connections from the application on Heroku. Since Heroku doesn't have static IP addresses, the [quotagaurd static add-on](https://elements.heroku.com/addons/quotaguardstatic) helps establish a connection with the remote database. For details on how this works, see [PR #91](https://github.com/fpdcc/ccfp-asset-dashboard/pull/91) and [PR #70](https://github.com/fpdcc/ccfp-asset-dashboard/pull/70), as well as discussions in issues [#59](https://github.com/fpdcc/ccfp-asset-dashboard/issues/59) and [#60](https://github.com/fpdcc/ccfp-asset-dashboard/issues/60).
-
-2. The Forest Preserves of Cook County's GIS database. We've setup a remote connection with the FPDCC's database. This connection also requires QuotaGuard Static.
+2. The Forest Preserves of Cook County's GIS database. We've setup a remote connection with the FPDCC's database. This connection also required QuotaGuard Static.
```

Comment:
I'm fairly confident in my updates for the rest of this section, but a bit unsure of this one. I couldn't quite find a clear answer in this repo or their AWS instance as to whether QuotaGuard was being used for the GIS database. So I've left it in as a record of what we used to do, for now.

I know that the CIP dashboard EC2 instances do not have an elastic ip (their term for a static ip), and I can somewhat assume that the RDS instance is connected automatically since they're both in AWS. But I wasn't able to find out how they connected the apps to the GIS db if that needs a static ip. If you have any insights, definitely let me know. Otherwise, maybe we can ask them in our next meeting.

Reply:
Let's definitely ask at our next meeting.